benchmarks/testdata/test-cmd-util.sh

```diff
@@ -655,7 +655,7 @@ run_pod_tests() {
   kube::test::get_object_assert rc "{{range.items}}{{$id_field}}:{{end}}" ''
   ## kubectl create --edit can update the label filed of multiple resources. tmp-editor.sh is a fake editor
   TEMP=$(mktemp /tmp/tmp-editor-XXXXXXXX.sh)
-  echo -e "#!/bin/bash\n$SED -i \"s/mock/modified/g\" \$1" > ${TEMP}
+  echo -e "#!/usr/bin/env bash\n$SED -i \"s/mock/modified/g\" \$1" > ${TEMP}
```

Comment:
It looks like the `sed` caught some testdata, which isn't desirable.

Let me try a  more focused `sed` command ... The continuous build will catch the broken tests for us

Reply:
Ah. I did see this change and it wasn't clear to me what the right choice was. Thank you.